lua/visual-whitespace.lua

```diff
@@ -88,13 +89,14 @@ local function get_marks(s_pos, e_pos, mode)
 
     -- process columns of current line
     repeat
-      adjusted_scol, _, match_char = string.find(line_text, "([ \t\r\n])", adjusted_scol)
+      adjusted_scol, _, match_char = string.find(line_text, "([ \160\t\r\n])", adjusted_scol)
 
       if adjusted_scol then
         if ff == 'dos' and line_len == adjusted_scol then
           table.insert(ws_marks, { cur_row, 0, CHAR_LOOKUP[match_char], "eol" })
         else
-          table.insert(ws_marks, { cur_row, adjusted_scol, CHAR_LOOKUP[match_char], "overlay" })
+		  local offset = match_char == "\160" and 1 or 0 
```

Comment:
I'm not very strong in Lua, so I couldn't solve the issue with the character offset. Could you suggest a more elegant solution for this?

Reply:
I found a more elegant solution! Apparently, `nbsp`s are two bytes, `\194\160`. If we match on `\194` instead of `\160`, we don't need the offset.